www/scripts/codecheckerviewer/codecheckerviewer.js

```diff
@@ -229,6 +239,35 @@ function (declare, topic, domConstruct, Dialog, DropDownMenu, MenuItem,
       runsTab.selectChild(runIdToTab[tabId]);
     });
 
+
+    topic.subscribe('openUserGuide', function () {
+      var that = this;
+
+      if (!this.userguide) {
+        this.userguide = new ContentPane({
+          title : 'User guide',
+          closable : true,
+          href  : 'userguide/doc/html/userguide.html',
```

Comment:
How does this handle if the user guide is missing because it failed to generate? The build script only issues a warning in this case.

Reply:
If the user guide is missing, dojo will handle it:
![userguide_missing](https://user-images.githubusercontent.com/6695818/30858277-4e84768e-a2bf-11e7-8e3c-05ba6957395a.png)